src/pages/index.js

```diff
@@ -16,6 +16,7 @@ const IndexPage = ({ data: { allEvent } }) => {
   const futureMeetups = sortedCities.filter(
     city => isFuture(city.node.info.date) || isToday(city.node.info.date)
   )
+  console.log(futureMeetups)
```

Comment:
Did you mean to leave this in here? 😁 ✂️

Reply:
oooops.  this is like my worst habit